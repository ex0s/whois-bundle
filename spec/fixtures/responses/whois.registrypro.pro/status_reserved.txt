The domain nii.pro is a Governmental Reserved Name and is available for registration only by eligible applicants.
