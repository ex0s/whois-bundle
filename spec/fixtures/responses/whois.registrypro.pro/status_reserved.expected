#status
  %s == :reserved

#reserved?
  %s == true
